src/bull/bull.service.ts

```diff
@@ -23,6 +23,12 @@ export class BullService {
     return this.queue.add('scrape-data', data);
   }
 
+  async sleepAtRandomInterval() {
+    const randomDelay = Math.floor(Math.random() * 5000) + 1000;
```

Comment:
It would be best to declare "magic" numbers (`5000`) as const:

```suggestion
    const INTERVAL_IN_SECONDS = 5000;
```